api/bases/test.openstack.org_tobikoes.yaml

```diff
@@ -51,7 +51,7 @@ spec:
                 type: string
               debug:
                 default: true
-                description: Run tests in parallel
+                description: Debug option
```

Comment:
maybe you could elaborate a bit more, in tempest's api definition we have much more detailed description:
https://github.com/openstack-k8s-operators/test-operator/blob/main/api/bases/test.openstack.org_tempests.yaml#L70

Reply:
I have removed the debug options as they are not used / not doing anything. I think adding them was most likely copy-paste error.